tools/diagnostic_format.py

```
@@ -0,0 +1,96 @@
+            print(result.stdout)
+        else:
+            print("❌ Erreur lors de la détection du type :")
+            print(result.stderr)
+    except Exception as e:
+        print(f"❌ Erreur ExifTool : {e}")
+        return
+    
+    # Test 2: Tenter d'écrire une métadonnée simple
+    try:
+        result = subprocess.run(
+            ['exiftool', '-Description=Test diagnostic', '-charset', 'utf8', str(file_path)],
```

Ruff (0.13.1)

Starting a process with a partial executable path

(S607)